spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/Binding.java

```diff
@@ -37,4 +39,17 @@
 	 * and a new Binding should be created instead.
 	 */
 	void unbind();
+
+	/**
+	 * Binds the target component represented by this instance. Should be used to call any
+	 * lifecycle method on the represented target. Implementations must be idempotent.
+	 * This method is called after the concrete {@link Binder} implementation returns from
+	 * its create binding methods
+	 */
+	void bind();
```

Comment:
Please see my comment related to the introduction of this method [here](https://github.com/spring-cloud/spring-cloud-stream/pull/1038/files#r132620289)